scripts/prow-smoke-test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+
+# Allow podman or docker 
+alias docker='podman'
```

Comment:
Since I don't have `podman` the script failed for me. We need to make sure it runs with both podman and docker. Maybe we can use a variable for this:
```sh
 if hash docker 2>/dev/null; then
        CONTAINER_ENGINE=docker
    elif hash podman 2>/dev/null; then
        CONTAINER_ENGINE=podman
    else
        echo >&2 "Container engine not installed. Install Docker or Podman and run the script again."
        exit 1
    fi
```

Reply:
And then replace the `podman run` commands with `$CONTAINER_ENGINE`.

Reply:
lol much better TY 😂